davenmathews/Bright_Events
# #153209109 Implement Event Response

## This pull request implements event response

This is on top of other previous commits

**The tasks completed are**
POST /api/v1/event/<int:event_id>/rsvp<int:response>

**This can be manually tested by:**
1. Clone the repository.
2. Use virtualenv to start a virtual environment.
3. Ensure the users.json and events.json are placed in your virtualenv folder and that they are empty
3. Install the requirements.txt.
4. Run appv1.py and open the local host in your browser.
5. Test the endpoints by registering, logging in and out, creating, modifying, and deleting an event.
6. Clicking on 'View event' will redirect you to a webpage where a single event is viewed
7. If logged in: User can respond by either confirming attendance or not else: One can only view the event's information.
8. On response, user should be added to guest list if attending, and to the rsvps list nonetheless.


**Constrains:**
Cannot access dashboard when user is not logged in
Will not modify event if the user in session is not the owner of the event
Will not delete event if the user in session is not the owner of the event
Will not display the response button if not logged in

**It Finishes #153209109 story on PT**
